views/partials/_whats-new.njk

```diff
@@ -6,7 +6,7 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-        <p class="govuk-body">6 September 2021: We’ve released GOV.UK Frontend v3.13.1. This ‘fix release’ contains bug fixes and minor visual improvements, particularly for users in forced colors mode and high contrast mode. <a href="https://github.com/alphagov/govuk-frontend/releases/tag/v3.13.1" class="govuk-link">Read the release notes to see what’s changed.</a>
+        <p class="govuk-body">30 September 2021: We’ve released GOV.UK Frontend v3.14.0. This ‘feature release’ contains bug fixes and additions to the spacing and alignment Sass functions. <a href="https://github.com/alphagov/govuk-frontend/releases/tag/v3.14.0" class="govuk-link">Read the release notes to see what’s changed.</a>
```

Comment:
`govuk-spacing` is a Sass function, but the alignment changes are just CSS classes – they're not functions and not really Sass related.

What about something like this? Too long?

```suggestion
        <p class="govuk-body">30 September 2021: We’ve released GOV.UK Frontend v3.14.0. This ‘feature release’ contains bug fixes, new override classes for text alignment, and a better way to add negative spacing using the govuk-spacing Sass function. <a href="https://github.com/alphagov/govuk-frontend/releases/tag/v3.14.0" class="govuk-link">Read the release notes to see what’s changed.</a>
```

Reply:
```suggestion
        <p class="govuk-body">30 September 2021: We’ve released GOV.UK Frontend v3.14.0. This ‘feature release’ contains bug fixes, new override classes for text alignment, and a better way to add negative spacing using the `govuk-spacing` Sass function. <a href="https://github.com/alphagov/govuk-frontend/releases/tag/v3.14.0" class="govuk-link">Read the release notes to see what’s changed.</a>
```

Reply:
@36degrees Thanks, what you're suggesting is way better than what was there before. I've just added back ticks around the function name. I'll commit this suggestion and then see in the preview whether it looks overlong.